attributes.yml

```diff
@@ -1,2 +1,2 @@
 curl_path: '/bin/curl'
```

Comment:
Can remove `curl_path` if not used in the test.

Reply:
removed attributes.yml

Reply:
re-added to avoid failure!